tests/traits/users.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
@CarlosLongarela - This file should not be added, was renamed to `usersHelper.php`